src/metaswitch/ellis/prov_tools/update_user.py

```diff
@@ -48,11 +48,11 @@ def main():
     parser.add_argument("-q", "--quiet", action="store_true", dest="quiet", help="don't display the user")
     parser.add_argument("--hsprov", metavar="IP:PORT", action="store", help="IP address and port of homestead-prov")
     parser.add_argument("--plaintext", action="store_true", help="store password in plaintext")
-    parser.add_argument("--ifc", metavar="iFC-FILE", action="store", dest="ifc_file", help="XML file containing the iFC")
+    parser.add_argument("--ifc", metavar="iFC-FILE", action="store", dest="ifc_file", help="XML file containing the iFC (default: iFC unchanged")
     parser.add_argument("--prefix", action="store", default="123", dest="twin_prefix", help="twin-prefix (default: 123)")
     parser.add_argument("dns", metavar="<directory-number>[..<directory-number>]")
     parser.add_argument("domain", metavar="<domain>")
-    parser.add_argument("--password", metavar="<password>")
+    parser.add_argument("--password", metavar="PASSWORD", help="new password (default: password unchanged)")
```

Comment:
I don't think you need `metavar="PASSWORD"` - [that's the default for positional arguments anyway](https://docs.python.org/2/library/argparse.html#metavar).